### PR TITLE
start add cities enum in province module

### PR DESCRIPTION
**Pull request by robatipoor**

### CODE REVIEW

File: src/province/mod.rs [hunk 1]
```diff
@@ -195,7 +194,7 @@ pub static PROVINCES: phf::Map<&'static str, Province> = phf_map! {
     },
 };
 
-#[derive(Debug, PartialEq, Eq, Hash, EnumString)]
+#[derive(Debug, PartialEq, Eq, Hash, EnumString,Display)]
```
Comment (omid) [inline src/province/mod.rs:197]: Code style. Please run `make check` to correct the styling also.

File: Cargo.toml [hunk 2]
```diff
@@ -18,5 +18,4 @@ lazy_static = "1.4.0"
 regex = "1.4.3"
 phf = { version = "0.8", features = ["macros"] }
 strum = { version = "0.20", features = ["derive"] }
```
Comment (omid) [inline Cargo.toml:20]: I'm thinking of adding a `nightly` feature to be able to release the library sooner 🤔 what do you think?

Comment (robatipoor) [reply Cargo.toml:20]: Firstly, our development speed is not high. Secondly, this library does not have many users yet, so I think it is not necessary

Comment (omid) [reply Cargo.toml:20]: So you mean we can release version 0.1.0?

**Review by omid — APPROVED** [review]

**Pull request merged**